src/StreamingZipReader/StreamingZipReader.cs

```diff
@@ -108,10 +112,22 @@ public async ValueTask<bool> MoveToNextEntryAsync(bool skipDirectories, Cancella
         return true;
     }
 
+    private static void ParseZIP64ExtraField(ReadOnlySpan<byte> span, out long compressedSize, out long uncompressedSize)
+    {
+        byte[] tag = [1, 0];
+        var tagIndex = span.IndexOf(tag);
```

Comment:
Instead of allocating a new array each time, this version will keep the byte sequence in a constant structure and point to it ([SharpLab](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA+ABATARgLABQhGKABALIAUASjAIYAmA8gHYA2AngMoAOdLAHmAcALjAB8pAM58WASkIBvQqVXTZAOgCSLBjARMAZpRr1m7brKGiJcgNoAGADSkcAXTkBuQgF8gA)):
```suggestion
        ReadOnlySpan<byte> tag = [1, 0];
        var tagIndex = span.IndexOf(tag);
```

I'm not thrilled that `IndexOf([1, 0])` fails to resolve the correct overload of IndexOf. I'll see what the language team thinks about this.

Reply:
Hmm yes I was slightly disappointed when `IndexOf([1, 0])` wouldn't compile. :(

Reply:
Hmm I don't understand what makes ReadOnlySpan a constant structure!

Reply:
I do now.. had to read up on how ReadOnlySpan actually works.